lib/config.js

```diff
@@ -48,7 +48,9 @@ export default class Config {
         baseDir: Joi.string(),
         ui: [Joi.bool(), Joi.object()],
         files: [Joi.string(), Joi.array()],
-        watchOptions: Joi.object(),
+        watchOptions: Joi.object().default().keys({
+          ignored: Joi.array().default(['node_modules'])
```

Comment:
I was thinking that if the user rewrote it, we'd rely on them to add back `node_modules`, but I'm sort of second guessing that now that you mention it. I'll just push `node_modules` and `outputDir` to it so that it's never watched.